packages/components/src/components/tile/tile.tsx

```diff
@@ -51,7 +51,7 @@ export class Tile extends LitElement implements InteractiveComponent, Selectable
   /**
    * When `true`, the component is active.
    *
-   * @deprecated
+   * @deprecated  in v2.8.0, removal target v6.0 - No longer necessary.
```

Comment:
Consistency suggestion:
```suggestion
   * @deprecated  in v2.8.0, removal target v6.0.0 - No longer necessary.
```